asyncorm/models/fields.py

```diff
@@ -157,7 +158,10 @@ class DateTimeField(Field):
     strftime = "%Y-%m-%d  %H:%s"
     args = ("auto_now", "choices", "db_column", "db_index", "default", "null", "strftime", "unique")
 
-    def serialize_data(self, value):
+    def sanitize_data(self, value):
```

Comment:
Field class have same method that returns the data without modification, thus we dont need to same placeholder (also other fields dont have same placeholder) until we will do not include NotImplementedError. But include NotImplemented is not neccesery, because we dont do anything with data in `serialize_data` method, but if you REALLY want i can add NotImplementedError and `serialize_data` method to subclasses